query-algebrizer/src/clauses/or.rs

```diff
@@ -0,0 +1,80 @@
+// Copyright 2016 Mozilla
```

Comment:
I'm not really reviewing this `Split cc.rs` commit -- do what you think makes sense.